Commonplace-Digital-Ltd/xyz
# COM-534 style by feature

## Description

Add style in postgres to get it by feature 
## Motivation and context



https://commonplace.atlassian.net/jira/software/c/projects/COM/boards/17?modal=detail&selectedIssue=COM-534&assignee=611c180cc2f3a50069e48b44

## How can this be tested?




- Run acorn
- Run xyz in this branch
- Go to testo2centremap
- Every feature in custom layer should have different colours and highlight colours 

## Screenshots (if appropriate):

## Types of changes




## Checklist:




- [ ] I have added at least 50% test coverage for my changes.
